Eastern-Research-Group/mywaterway
# Bugfix/120 protect tab unavailable

## Related Issues:
* https://jira.epa.gov/browse/HMW-120

## Main Changes:
* HMW-120 Fixed issue of the entire protect tab being unavailable when the attains services are down.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/overview
2. Click on the protect tab
3. Verify all of the accordions load correctly.
4. Open dev tools
5. Block all network requests coming from the `attains.epa.gov` domain
6. Navigate to http://localhost:3000/community/dc/overview
7. Click on the protect tab
8. Verify all of the accordions load correctly.
